setupext.py

```diff
@@ -1380,6 +1380,30 @@ def get_install_requires(self):
             return []
 
 
+class Subprocess32(SetupPackage):
```

Comment:
Does deriving from `SetupPackage` make it optional (as it states in the message below)?